sub-sites/wiki/content/developers/build.md

```diff
@@ -7,21 +7,15 @@ In order to set up or build the source code some of the projects from the yukino
 Requirements:
 - [Flutter SDK](https://docs.flutter.dev/get-started/install)
 - [Android SDK](https://developer.android.com/studio)
-- (Optional)[Github Desktop](https://desktop.github.com/)
 - (Optional)[Git](https://git-scm.com/downloads)
 
-To begin, you have three options for getting source:
+To begin, you have two options for getting source:
 Option one:
-In your browser, navigate to the [Kazahana repo](https://github.com/yukino-org/kazahana) and press the green code button and download the zip.
-Option two(Github Desktop):
-In your browser, navigate to the [Kazahana repo](https://github.com/yukino-org/kazahana) and press the green code button and open with Github dekstop. 
-Option three(Git):
-Open a new terminal and run the following command
-```
-git clone https://github.com/yukino-org/kazahana.git
-```
+In your browser, navigate to the [Kazahana Repository](https://github.com/yukino-org/kazahana) and press the green code button and download the zip.
```

Comment:
Remove this too, I'm pretty sure it's a hassle to work without Git. Some scripts might use Git internally in the future.